packages/fether-react/src/utils/withHealth.js

```diff
@@ -27,13 +27,16 @@ const electron = isElectron() ? window.require('electron') : null;
 // List here all possible states of our health store. Each state can have a
 // payload.
 export const STATUS = {
-  CLOCKNOTSYNC: Symbol('CLOCKNOTSYNC'), // Local clock is not sync
+  NO_NODE_CONNECTED_AND_NO_INTERNET: Symbol(
+    'NO_NODE_CONNECTED_AND_NO_INTERNET'
+  ),
+  NO_CLOCK_SYNC: Symbol('NO_CLOCK_SYNC'), // Local clock is not sync
   DOWNLOADING: Symbol('DOWNLOADING'), // Currently downloading Parity
-  GOOD: Symbol('GOOD'), // Everything's fine
-  NOINTERNET: Symbol('NOINTERNET'), // No Internet connection
-  NOPEERS: Symbol('NOPEERS'), // Not connected to any peers
   LAUNCHING: Symbol('LAUNCHING'), // Parity is being launched (only happens at startup)
-  SYNCING: Symbol('SYNCING') // Obvious
+  NODE_CONNECTED_AND_NO_INTERNET: Symbol('NODE_CONNECTED_AND_NO_INTERNET'),
```

Comment:
Why can't we be more atomic: `NODE_CONNECTED` an `NO_INTERNET`: the 2 here can be derived from those 2 simpler ones.

EDIT: withHealth right now returns only one of the STATUS here. Maybe we could return an array of STATUS, it might make more sense, what do you think @ltfschoen ?

Reply:
Totally agree, I'll make that change

Reply:
Actually, I think it'll be better to just use an object with boolean property values

Reply:
Update: I decided to create a `status` object containing each of the different states whose value is boolean